src/routes/copy-configuration-extended/utils.js

```diff
@@ -48,8 +48,11 @@ export const sendReportOnWarnings = (errors) => {
 }
 
 export const handleCheckboxChange = (dispatch, checkbox, value, siteId, setConfigurationStatus) => {
+  const checkBoxId = checkbox.id
```

Comment:
Instead of adding the show/hide logic in here, do it in `services` in `recaptchaOptions.js`.
Similar to what we have in `socialOptions.js`:
<img width="1259" alt="image" src="https://github.com/user-attachments/assets/cee6412e-69f2-4b19-b6c2-8ef9f68755a8">